exercises/ex3.cpp

```diff
@@ -1,43 +1,37 @@
 /* Could you still use a switch case here? May you can use a map. */
 
+#include <map>
 #include <iostream>
+
 using namespace std;
 
+string get_name()
+{
+  string input;
+  cout << "Enter a famous name+surname, ex. BarackObama " << endl;
+  cin >> input;
+  return input;
+}
+
+void check_description(map<string, string> map, string name)
+{
+  cout << (map.find(name) != map.end() ? map[name] : "Invalid input! Please enter a good name!") << endl;
+}
+
 int main()
 {
-  string textInput;
+  string textInput = get_name();
```

Comment:
potresti passare per riferimento textInput ed evitare di creare una variabile all'interno di `get_name()`, inoltre, usa il camelCase per i nomi delle funzioni e delle variabili, lo snake_case si usa in python
https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats